ci/report_result.yml

```diff
@@ -4,6 +4,7 @@
     cmd: |
       cp  {{ fvt_dir }}/test_run_result.out {{ logs_dir }}/test_run_result.log
       cp  {{ fvt_dir }}/summary_results.log {{ logs_dir }}/summary_results.log
+      cp  ~/.ansible.log/*.xml {{ logs_dir }}/junittest_run_result.xml
```

Comment:
This is unrelated to updating the autoscaling test
```suggestion
```

Reply:
https://github.com/infrawatch/feature-verification-tests/pull/136

Reply:
Since we have the [alternative junit plugin PR](https://github.com/infrawatch/feature-verification-tests/pull/132), the configuration of the junit plugin should be removed from this PR, to keep the scope limited to a single change (i.e. updating the autoscaling test)